lib/src/utils/duration.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+
+import 'package:lichess_mobile/src/utils/l10n_context.dart';
+
+extension DurationExtensions on Duration {
+  String toDaysHoursMinutes(BuildContext context) {
```

Comment:
perhaps this function could have a unit test?

Reply:
Of course! It looks like it can fail in lots of disastrous ways...

Reply:
This method works for english, but I'm not 100% sure it will work for every language out there...
Although now that I think about it, this could be solved by checking `Intl.getCurrentLocale()`, and if there's a special case, just alter the function behaviour.

Should I add this as a comment in case someone in the future stumbles with this problem?